login.go

```diff
@@ -0,0 +1,31 @@
+package keycloak
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+	"gopkg.in/h2non/gentleman.v2"
+)
+
+// InitiateLogin begins the login flow
+func (c *Client) InitiateLogin(realmName string, loginURLEncoded InitatePKCELogin) (*http.Response, error) {
+	var req *gentleman.Request
+	{
+		var authPath = fmt.Sprintf("/auth/realms/%s/protocol/openid-connect/auth", realmName)
```

Comment:
pull this up like https://github.com/tozny/keycloak-client/blob/master/clients.go#L13

Reply:
Since i have to use sprintf, I added it as a const! but I cant URL param replace it like the example in clients